From 9c5a7483556f460300518763dd9ff78fc96c996f Mon Sep 17 00:00:00 2001
From: Marian Chereji <marian.chereji@freescale.com>
Date: Mon, 10 Dec 2012 17:04:40 +0200
Subject: [PATCH 086/518] fmd: Export FrameReplicator API

Added exports for the FrameReplicator API:

FM_PCD_FrmReplicSetGroup
FM_PCD_FrmReplicDeleteGroup
FM_PCD_FrmReplicAddMember
FM_PCD_FrmReplicRemoveMember

Signed-off-by: Marian Chereji <marian.chereji@freescale.com>
[Grabbed from the branch, LINUX_IR5.2.0, of
https://git.freescale.com/git-private/cgit.cgi/ppc/alu-b4860/linux.git.]
Signed-off-by: Tiejun Chen <tiejun.chen@windriver.com>
---
 .../dpa/NetCommSw/src/inc/wrapper/lnxwrp_exp_sym.h |    6 ++++++
 1 files changed, 6 insertions(+), 0 deletions(-)

diff --git a/drivers/net/ethernet/freescale/dpa/NetCommSw/src/inc/wrapper/lnxwrp_exp_sym.h b/drivers/net/ethernet/freescale/dpa/NetCommSw/src/inc/wrapper/lnxwrp_exp_sym.h
index bfa427e..4c60893 100644
--- a/drivers/net/ethernet/freescale/dpa/NetCommSw/src/inc/wrapper/lnxwrp_exp_sym.h
+++ b/drivers/net/ethernet/freescale/dpa/NetCommSw/src/inc/wrapper/lnxwrp_exp_sym.h
@@ -104,6 +104,12 @@ EXPORT_SYMBOL(FM_PCD_ManipNodeSet);
 EXPORT_SYMBOL(FM_PCD_ManipNodeDelete);
 EXPORT_SYMBOL(FM_PCD_ManipGetStatistics);
 EXPORT_SYMBOL(FM_PCD_ManipNodeReplace);
+#if (DPAA_VERSION >= 11)
+EXPORT_SYMBOL(FM_PCD_FrmReplicSetGroup);
+EXPORT_SYMBOL(FM_PCD_FrmReplicDeleteGroup);
+EXPORT_SYMBOL(FM_PCD_FrmReplicAddMember);
+EXPORT_SYMBOL(FM_PCD_FrmReplicRemoveMember);
+#endif /* DPAA_VERSION >= 11 */
 
 #ifdef FM_CAPWAP_SUPPORT
 EXPORT_SYMBOL(FM_PCD_StatisticsSetNode);
-- 
1.7.5.4

